bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue

```diff
@@ -383,6 +394,7 @@ export default {
           }).open()
         })
       })
+        .catch(() => {})
```

Comment:
Is this really required? I don't think so.

Reply:
As discussed offline, catch is required to suppress ugly promise.reject message but I don't like catch-alls either. I added a specific reject-reason and cleanly catch it and rethrow everything else now.